.github/workflows/terraform.yaml

```diff
@@ -1,22 +1,59 @@
 name: Validate Terraform
+
 on:
+  - push
```

Comment:
Curious why you enabled this?

Reply:
It's so the checks run whenever you push your current work up to a branch whether you PR or not.
It also causes the checks to run following a merge (which I think really has a "silent push" of the merged code).

Reply:
I see, that makes sense.  Thanks.